src/components/home/TimeTracker.js

```diff
@@ -32,11 +32,33 @@ export const GrowthHours = ({ hoursUsed, hoursRemaining, totalGrowth }) => {
 }
 
 const TimeTracker = () => {
-  const { billable, growth } = EmployeeTimeTracking
+  const initialState = {
+    billable: {
+      currentHours: 0,
+      currentTarget: 0,
+      totalTarget: 0,
+    },
+    growth: {
+      hoursUsed: 0,
+      hoursRemaining: 0,
+      totalGrowth: 0,
+    },
+  }
+
+  const [data, setData] = useState(initialState)
+
+  useEffect(() => {
+    API.get("http://localhost:5000/Employee/")
+      .then(res => {
+        setData(res.data)
+      })
+      .catch(ex => console.log(ex))
+  }, [])
```

Comment:
Can we just use the useAPI hook for this logic? or a modified version of it

Reply:
I'm tempted to steer away from the API hook because it's pretty specific and might not fit all use cases. We should talk about this as a group and bring in the Redux discussion.

Reply:
using a useEffect hook in the meantime